src/routes/startup_profile.js

```diff
@@ -0,0 +1,13 @@
+const { mockCollection } = require('../../database/schema.js');
+
+module.exports = (req, res) => {
+  const id = req.params.id;
+  mockCollection.findOne({_id:id}, (error, startup)=> {
+    if(error) res.render(error);
```

Comment:
So this is the case where someone clicks on a startup logo/name on the results page and for some reason, we won't find it in our database. 
in this case, we should handle the error in a better way. rendering 'error' won't be the best user experience.